web/themes/joinup/joinup_theme.theme

```diff
@@ -39,24 +42,35 @@ function joinup_theme_preprocess_page(&$variables) {
   // Set page title.
   $title = \Drupal::service('title_resolver')->getTitle($request, $route_match->getRouteObject());
 
+  // Check if page is form page and page content has no white background.
+  $is_form_container = FALSE;
+
   // We need to show the header in a blue variant on certain routes:
   // node and rdf entity forms, user forms.
   $route_name = $route_match->getRouteName();
   $blue_header_routes = [
+    'adms_validator.adms_validator_form',
     'collection.propose_form',
+    'contact_form.contact_page',
     'entity.node.edit_form',
     'entity.rdf_entity.edit_form',
-    'node.add',
-    'node.add_page',
     'rdf_entity.propose_form',
-    'rdf_entity.rdf_add',
```

Comment:
When I created this ticket I was thinking about all forms which are in use. Path like rdf_entity/add/asset_release are not in the normal site workflow, it is not even possible to use them (there are errors). I don't know why this "Add RDF" button exist, some of the forms there cannot be submitted.